crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1226,20 +1274,6 @@ impl actor::HcP2p for HolochainP2pActor {
         Box::pin(async move {
             use crate::types::event::HcP2pHandler;
 
-            // This single function is/was a weird mix of a bunch of strange
-            // operations. We're trying to narrow it down now.
-            //
-            // - `reflect_ops` back to holochain
-            //   - TODO - remove this (fix anything it breaks)
```

Comment:
Keep this TODO?

Reply:
I assumed it was just for the integration and it didn't seem like there were any tests that needed that functionality. I can't imagine what it would have been for